src/coreclr/src/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Windows.cs

```diff
@@ -6,9 +6,27 @@ namespace System.Globalization
 {
     internal static partial class GlobalizationMode
     {
+        internal static bool UseIcu { get; private set; } = GetUseIcuMode();
```

Comment:
```suggestion
        internal static bool UseIcu { get; } = GetUseIcuMode();
```
I believe having a setter on this will prevent the field from being readonly, that in turn prevents optimizations.

Reply:
Ah right, that was a leftover from the initial way I had implemented this.